src/styles/index.scss

```diff
@@ -1,3 +1,75 @@
 body {
+  padding: 0;
   margin: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  position: relative;
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the border should be formatted as `border: border-width border-style border-color;`. Consider splitting the shorthand property into its individual components.